src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_main_library.baseline

```diff
@@ -1070,7 +1070,7 @@ LibraryServiceClient.prototype.deleteBook = function(request, options, callback)
  * @param {Object=} request.updateMask
  *   A field mask to apply, rendered as an HTTP parameter.
  *
- *   This object should have the same structure as [google.protobuf.FieldMask]{@link external:"google.protobuf.FieldMask"}
+ *   This object should have the same structure as [FieldMask]{@link FieldMask}
```

Comment:
This change should be fine @jmdobry, correct?

Reply:
Yes, and necessary. However, it should be `[FieldMask]{@link google.protobuf.FieldMask}`